### DISCUSSION TITLE
Extend DistributedLossWrapper to use with ref_emb and ref_labels

### DISCUSSION THREAD

**Discussion by Bomme** [KevinMusgrave/pytorch-metric-learning#519] [Ideas]
Hi,
first of all thanks for this amazing library!
Do you have any plans for extending the functionality of `DistributedLossWrapper` for it to be used with `ref_emb` and `ref_labels`?

**Comment by KevinMusgrave** [comment]
Yes, I just need to review and merge this code: https://github.com/KevinMusgrave/pytorch-metric-learning/pull/503

**Reply by Bomme** [reply]
Thanks for the quick reply! I missed that PR! (I only checked issues and discussions)

**Reply by KevinMusgrave** [reply]
This feature is now available in v1.6.0

**Reply by Bomme** [reply]
Many thanks!
